build.rs

```diff
@@ -245,6 +249,10 @@ fn check_php_version(info: &PHPInfo) -> Result<()> {
         println!("cargo:rustc-cfg=php83");
     }
 
+    if version >= PHP_84_API_VER {
+        println!("cargo:rustc-cfg=php84");
+    }
+
```

Comment:
in #334 i touched this as well, as having the cfg's set up this way didn't allow to detect specific versions. I think we should have a cfg only set if it is exactly that version of php.

https://github.com/davidcole1340/ext-php-rs/pull/334/files#diff-d0d98998092552a1d3259338c2c71e118a5b8343dd4703c0c7f552ada7f9cb42R243

Reply:
Yeah, I think in this case it's actually what we want, as all future versions of PHP will want the changes in this PR, however agree that these fgs are a bit misleading as "php82" means "8.2 and higher"

Reply:
true